### ISSUE TITLE
please scrub out openmemoryinterface dot org links

### ISSUE THREAD

**Issue by TanjIsGray** [OpenCAPI/omi_host_fire#6]
OpenMemoryInterface domain registration lapsed and has been captured by a malware host.  Be careful how you verify that...

I suggest you scrub all references to that site.  The only place documentation now exists on the web is OpenCAPI.

Best regards,
Tanj

**Comment by rajhlinux** [comment]
What happened to OMI, is it true that they sold out their souls to CXL? 
Thats the most pathetic shady thing ever, OMI is way better than CXL, offers lower latency and uses serial termination.

So as of 2026, there is no way to use OMI, I cheaked OpenCAPI's website, the site looks like it was designed by a child, everything is so disorganized and confusing.

Are there any PCB design schematics or references to use OMI? Are there any alternatives to OMI (not CXL)?

Thanks.

**Comment by TanjIsGray** [comment]
No, they just passed the specs to OCP but OCP did nothing to maintain the website.

Take a look at Credo omniconnect and Weaver.  It is a DDIMM for modern systems.  Yes it is a stripped down CXL FLIT (whatever that means in practice) but it carries AXI semantics instead of CXL so you still might find it palatable.  There was an EETimes article on it last month which was worth reading.
